src/org/parosproxy/paros/extension/report/ReportGenerator.java

```diff
@@ -126,7 +127,7 @@ public static File stringToHtml(String inxml, String infilexsl, String outfilena
 					bw = new BufferedWriter(new FileWriter(outfilename + "-orig.xml"));
 					bw.write(inxml);
 				} catch (IOException e2) {
-					logger.error(e.getMessage(), e);
+					logger.error(Constant.messages.getString("report.write.diskspace.error.log.message"), e);

```

Comment:
From a usability perspective should we also pop a message dialog here? There is a small chance that a huge report might be written where the 5MB warning wouldn't be seen...